src/python.yml

```diff
@@ -3,7 +3,13 @@
   name: Install pip3/python3 and remove pip2/python2
   become: yes
   become_method: sudo
-  roles:
-    - pip
-    - python
-    - remove_python2
+  tasks:
+    - name: Install pip3
+      ansible.builtin.include_role:
+        name: pip
+    - name: Install Python 3
+      ansible.builtin.include_role:
+        name: python
```

Comment:
Thoughts on flipping these now that they're no longer a list?

Reply:
Please see commit 2c1b7a1.

Reply:
Can you please add a comment here similar to (or the same as) the commit message you wrote in https://github.com/cisagov/skeleton-packer/commit/2c1b7a1269f28c6488380550e87c6658b7ea6a5e?  That will be a helpful reminder to us and others why the order matters here.

Reply:
Looks great, thanks!